### DISCUSSION TITLE
Submission Error

### DISCUSSION THREAD

**Discussion by bluetex315** [PaddlePaddle/PaddleSpatial#255] [BaiduKDDCup2022]
Hi,

could you please help check why there was an error about our last submission? The team name is zealot and the submission time is 2022-07-12 16:27. It throws `no job log` 
<img width="392" alt="image" src="https://user-images.githubusercontent.com/86997509/178491009-93220db9-349f-4554-86c7-0f107459bc60.png">

Thanks a lot :)

**Comment by bluetex315** [comment]
sorry the team name is zealen. Very much appreciate it!

**Comment by ramber1836** [comment]
As suggested by the message, the log is not available.

**Reply by bluetex315** [reply]
Could you please hint more on that? Does that mean there exists some part of the code prevents prevent the log? We indeed had a Logger class in the predict.py, is that a major problem because maybe you are also relying on the Logger to provide details?

**Reply by ramber1836** [reply]
This is probably because the evaluation is terminated by the system.
